packages/dds/tree/src/shared-tree/treeView.ts

```diff
@@ -40,7 +38,18 @@ export interface FlexTreeView<in out TRoot extends FlexFieldSchema> extends IDis
 	 * This is a non-owning reference: disposing of this view does not impact the branch.
 	 */
 	readonly checkout: ITreeCheckout;
+}
 
+/**
+ * An editable view of a (version control style) branch of a shared tree.
+ * @privateRemarks
+ * TODO:
+ * 1. Once ISharedTreeView is renamed this can become ISharedTreeView.
```

Comment:
Note this somewhat newish guidance around interface naming: https://github.com/microsoft/FluidFramework/wiki/Coding-Guidelines#-do-not-prefix-interfaces-with-i

Just as a heads-up. We're not trying to do an overhaul of existing names, but for new types / when we are doing renames anyways, good to keep in mind.